tests/contracts/error/Xargo.toml

```diff
@@ -0,0 +1,5 @@
+[dependencies]
+core = { default-features = false, features = ["panic_immediate_abort"] }  
+
+[dependencies.compiler_builtins]
+stage = 1
```

Comment:
Same here.

Also, if we want to use Xargo maybe we should document it and set the Makefile / travis in order to use it. At the moment the Xargo.toml is irrelevant since there is nothing in the build pipeline that uses it, and it's a bit misleading.
We either remove it or decided to introduce it. If we introduce it we should document it why.

Reply:
this should be a separate pr

Reply:
Unless we remove it, agreed. If you want to keep it for the time being then, just file a new issue about Xargo and add the issue reference here as comment so we can keep track in the future. Thanks!